aiohttp/http_writer.py

```diff
@@ -62,6 +70,9 @@ def _write(self, chunk):
         writer can't be used after write_eof() method being called.
         write() return drain future.
         """
+        if self._on_chunk_sent:
```

Comment:
Please do check `self._on_chunk_sent is not None`. It is more idiomatic.

Reply:
changed